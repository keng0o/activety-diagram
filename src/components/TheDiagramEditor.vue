<template>
  <div>
    <p>hello diagram editor</p>
  </div>
</template>

<script>
export default {
  name: "the-diagram-editor"
};
</script>
