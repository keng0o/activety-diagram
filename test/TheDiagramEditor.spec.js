import { shallow, createLocalVue } from "vue-test-utils";
import ace from "../static/ace/ace.js";
import TheDiagramEditor from "../src/components/TheDiagramEditor.vue";

describe("TheDiagramEditor component", () => {
  it("render", () => {
    expect(shallow(TheDiagramEditor).html()).toMatchSnapshot();
  });
});
