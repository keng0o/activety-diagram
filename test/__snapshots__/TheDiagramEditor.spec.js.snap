// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheDiagramEditor component render 1`] = `"<div><div class=\\"editor is-side ace_editor ace-tm ace_focus\\"><textarea class=\\"ace_text-input\\" wrap=\\"off\\" autocorrect=\\"off\\" autocapitalize=\\"off\\" spellcheck=\\"false\\" style=\\"opacity: 0; position: fixed; top: 0px;\\"></textarea><div class=\\"ace_gutter\\"><div class=\\"ace_layer ace_gutter-layer\\"></div><div class=\\"ace_gutter-active-line\\"></div></div><div class=\\"ace_scroller\\"><div class=\\"ace_content\\"><div class=\\"ace_layer ace_print-margin-layer\\"><div class=\\"ace_print-margin\\" style=\\"left: 4px; visibility: hidden;\\"></div></div><div class=\\"ace_layer ace_marker-layer\\"></div><div class=\\"ace_layer ace_text-layer\\" style=\\"padding: 0px 4px;\\"></div><div class=\\"ace_layer ace_marker-layer\\"></div><div class=\\"ace_layer ace_cursor-layer\\"><div class=\\"ace_cursor\\"></div></div></div></div><div class=\\"ace_scrollbar ace_scrollbar-v\\" style=\\"display: none; width: 20px;\\"><div class=\\"ace_scrollbar-inner\\" style=\\"width: 20px;\\"></div></div><div class=\\"ace_scrollbar ace_scrollbar-h\\" style=\\"display: none; height: 20px;\\"><div class=\\"ace_scrollbar-inner\\" style=\\"height: 20px;\\"></div></div><div style=\\"height: auto; width: auto; top: 0px; left: 0px; visibility: hidden; position: absolute; white-space: pre; overflow: hidden;\\"><div style=\\"height: auto; width: auto; top: 0px; left: 0px; visibility: hidden; position: absolute; white-space: pre; overflow: visible;\\"></div><div style=\\"height: auto; width: auto; top: 0px; left: 0px; visibility: hidden; position: absolute; white-space: pre; overflow: visible;\\">XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX</div></div></div></div>"`;
